test/__snapshots__/index.test.js.snap

```diff
@@ -769,14 +769,24 @@ exports[`Validation should fail validation for integer with exclusive minimum 2`
 
 exports[`Validation should fail validation for integer with minimum 1`] = `
 "Invalid configuration object. Object has been initialised using a configuration object that does not match the API schema.
- - configuration.integerWithMinimum should be a integer (should be >= 5, should be <= 20)."
+ - configuration.integerWithMinimum should be a integer (should be >= 5 and <= 20)."
 `;
 
 exports[`Validation should fail validation for integer with minimum and maximum 1`] = `
 "Invalid configuration object. Object has been initialised using a configuration object that does not match the API schema.
  - configuration.integerWithMinimum should be >= 5."
 `;
 
+exports[`Validation should fail validation for integer with not minimum 1`] = `
+"Invalid configuration object. Object has been initialised using a configuration object that does not match the API schema.
+ - configuration.integerNotWithMinimum should not be integer | integer (should be < 5)"
+`;
+
+exports[`Validation should fail validation for integer with not minimum and maximum 1`] = `
+"Invalid configuration object. Object has been initialised using a configuration object that does not match the API schema.
+ - configuration.integerWithNotMinMax should not be integer | integer (should be < 5 or > 20)"
```

Comment:
Some misleading

Reply:
nope it is correct. It could be any non-integer or integer within range

Reply:
So better write: `should any non-integer | integer (should be < 5 or > 20)`